BUILD.md

```diff
@@ -0,0 +1,170 @@
+# Build FreeOrion
+
+
+## Hardware and OS Requirements
+
+FreeOrion will compile on most recent Windows, Mac OS X and Linux operating
```

Comment:
I think with Windows and OSX we can be a bit more specific: Windows: Windows 7 (64bit) or later, Mac: OSX 10.10 or later.